CorpusSearch/ClientApp/src/App.tsx

```diff
@@ -18,8 +19,9 @@ export const App = () => {
               <Routes>
                   <Route path='/' element={<Home key={k}/>} />
                   <Route path='/docs/:docId' element={<DocumentView/>} />
+                  <Route path={"bit"} element={<BitPlayer/>} />
```

Comment:
So we can extend functionality here without changing the URL

```suggestion
                  <Route path={"/tools/youtube"} element={<BitPlayer/>} />
```